docs/source/install.rst

```diff
@@ -6,87 +6,56 @@ Installation Guide
 Recommended Environments
 ------------------------
 
-We recommend these Linux distributions.
+We recommend the following Linux distributions.
 
-* `Ubuntu <https://www.ubuntu.com/>`_ 14.04/16.04 LTS 64bit
-* `CentOS <https://www.centos.org/>`_ 7 64bit
-
-The following versions of Python can be used: 2.7.6+, 3.4.3+, 3.5.1+, and 3.6.0+.
-
-.. warning::
-
-   If you are using certain versions of conda, it may fail to build CuPy with error
-   ``g++: error: unrecognized command line option ‘-R’``.
-   This is due to a bug in conda (see `conda/conda#6030 <https://github.com/conda/conda/issues/6030>`_ for details).
-   If you encounter this problem, please downgrade or upgrade it.
+* `Ubuntu <https://www.ubuntu.com/>`_ 14.04 / 16.04 LTS (64-bit)
+* `CentOS <https://www.centos.org/>`_ 7 (64-bit)
 
 .. note::
 
-   We are testing CuPy automatically with Jenkins, where all the above *recommended* environments are tested.
+   We are automatically testing CuPy on all the recommended environments above.
    We cannot guarantee that CuPy works on other environments including Windows and macOS, even if CuPy looks running correctly.
 
-CuPy uses C++ compiler such as g++.
-You need to install it before installing CuPy.
-This is typical installation method for each platform::
-
-  # Ubuntu 14.04
-  $ apt-get install g++
 
-  # CentOS 7
-  $ yum install gcc-c++
-
-If you use old ``setuptools``, upgrade it::
-
-  $ pip install -U setuptools
-
-
-Dependencies
+Requirements
 ------------
 
-Before installing CuPy, we recommend to upgrade ``setuptools`` if you are using an old one::
-
-  $ pip install -U setuptools
+You need to have the following components to use CuPy.
 
-The following Python packages are required to install CuPy.
-The latest version of each package will automatically be installed if missing.
+* `NVIDIA CUDA GPU <https://developer.nvidia.com/cuda-gpus>`_
+    * Compute Capability of the GPU must be at least 3.0.
+* `CUDA Toolkit <https://developer.nvidia.com/cuda-zone>`_
+    * Supported Versions: 7.0, 7.5, 8.0, 9.0 and 9.1.
+    * If you have multiple versions of CUDA Toolkit installed, CuPy will choose one of the CUDA installation automatically.
+      See :ref:`install_cuda` for details.
+* `Python <https://python.org/>`_
+    * Supported Versions: 2.7.6+, 3.4.3+, 3.5.1+, and 3.6.0+.
```

Comment:
How about removing the Oxford comma, for consistency. This is indeed a small detail...